compiler/noirc_evaluator/src/brillig/brillig_ir/codegen_intrinsic.rs

```diff
@@ -66,48 +66,45 @@ impl<F: AcirField + DebugToString> BrilligContext<F> {
     pub(crate) fn codegen_to_radix(
         &mut self,
         source_field: SingleAddrVariable,
-        target_vector: BrilligVector,
+        target_array: BrilligArray,
         radix: u32,
-        limb_count: usize,
         big_endian: bool,
         limb_bit_size: u32,
     ) {
         assert!(source_field.bit_size == F::max_num_bits());
 
-        self.usize_const_instruction(target_vector.size, limb_count.into());
-        self.usize_const_instruction(target_vector.rc, 1_usize.into());
-        self.codegen_allocate_array(target_vector.pointer, target_vector.size);
+        let size = SingleAddrVariable::new_usize(self.allocate_register());
+        self.usize_const_instruction(size.address, target_array.size.into());
+        self.usize_const_instruction(target_array.rc, 1_usize.into());
+        self.codegen_allocate_array(target_array.pointer, size.address);
```

Comment:
```suggestion
self.codegen_allocate_fixed_length_array(target_array.pointer, target_array.size);
self.usize_const_instruction(target_array.rc, 1_usize.into());
```
I think you can replace with this, then you don't need to keep `size` around. `usize_const_instruction` allocates a register internally.

Reply:
We'd need to initialise another constant register in order to hold the array size for the for-loop so it's best to have this explicit so we can avoid the extra `CONST` opcode imo.